scripts/west_commands/runners/spi_burn.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021, Telink Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import time
+import sys
+import subprocess
+
+from runners.core import ZephyrBinaryRunner, RunnerCaps
+
+TELINK_TOOLCHAIN_BASE = os.environ["TELINK_TOOLCHAIN_BASE"]
```

Comment:
This seems like it is not needed, and it will cause `_import_runner_module` to issue a warning for this runner whenever the environment variable is not defined, even when the user is not flashing a b91 board.

Most runners just rely on `PATH` to find the executables they are looking for; why should this one be different?